openshift/console
# OCPBUGS-15310: Helm Chart installation modal "Documentation" field is always N/A

**Fixes**: 
https://issues.redhat.com/browse/OCPBUGS-15310

**Analysis / Root cause**: 
For Helm chart, there is no specific place where documentation is defined. It was always N/A

**Solution Description**: 
Disabled Documentation in catalog details page for HelmChart

**Screen shots / Gifs for design review**: 

-----BEFORE-----


https://github.com/openshift/console/assets/102503482/b8b39440-1b34-4e01-86e5-78e7d6c1517a



----AFTER-------


https://github.com/openshift/console/assets/102503482/a5b07de0-7113-4b56-8e9d-52c274d72f72




**Unit test coverage report**: 
NA

**Test setup:**

1. Go to Helm chart catalog
2. View any chart
3. See documentation = "N/A"

**Browser conformance**:
